### ISSUE TITLE
Disable minify pixel in production environment

### ISSUE THREAD

**Issue by nicolazirilli** [vtex-apps/store-discussion#521]
I'm having problems with my pixel app (fanplayr-pixel) when it is installed in production.

In fact, I see the code inside head.html minified, but the minified result is messed up with many problems.

I see that the google-tag-manager-pixel is not minified when installed, so I would like to understand how I can disable the minified result of my pixel. Looking at all configuration files I don't see any difference between my project and GTM pixel project.

Thank you

**Comment by vitorflg** [comment]
Hi @nicolazirilli. We have just released a fix for this problem. Can you check if everything is ok now?

Today, we do not have an option to skip minification but we enabled it on development workspaces for debugging purposes.

**Comment by nicolazirilli** [comment]
@vitorflg I installed the fix and nothing changed.

To be clear, this is my code inside head.html:

![image](https://user-images.githubusercontent.com/77513290/105158528-59d91200-5b0e-11eb-9b96-6c6c3f8ade80.png)

This is the minified result that I see on the production environment (after I beautified it to make it readable):
![image](https://user-images.githubusercontent.com/77513290/105158655-82610c00-5b0e-11eb-92d0-aff1f524a48b.png)

A portion of the code is completely disappeared (the part that registers the service worker).

What is going on?

Finally, how I could enable modification on dev env? It is frustrating trying to find a solution continuing to publish and install in production to see the final result.

Thank you
